normalizer/tests/test_api_comparison.py

```
@@ -0,0 +1,218 @@
+        
+        print()
+        
+        # Jageocoder API
+        print("【Jageocoder API】")
+        jageocoder_url = API_ENDPOINTS["jageocoder"]
+        jageocoder_params = {"addr": address}
+        try:
+            jageocoder_res = requests.get(jageocoder_url, params=jageocoder_params, timeout=10)
+            jageocoder_data = jageocoder_res.json()
+            print(f"  エンドポイント: {jageocoder_url}")
+            print(f"  レスポンス（最初の1件）:")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)